src/functions-reference/matrix_operations.Rmd

```diff
@@ -958,6 +976,112 @@ v)` (and similarly for pre-multiplication). By the same token, it is
 better to use `quad_form_diag(m, v)` rather than `quad_form(m,
 diag_matrix(v))`.
 
+<!-- matrix; identity_matrix; (int k); -->
+\index{{\tt \bfseries identity\_matrix\_matrix }!{\tt (int k): matrix}|hyperpage}
+
+`matrix` **`identity_matrix`**`(int k)`<br>\newline
+Create an identity matrix of size $k \times k$
+
+
+
+## Container construction functions {#container-construction}
+
+<!-- real[]; linspaced_array; (int n, real lower, real upper); -->
+\index{{\tt \bfseries linspaced\_array }!{\tt (int n, real lower, real upper): real[]}|hyperpage}
+
+`real[]` **`linspaced_array`**`(int n, real lower, real upper)`<br>\newline
+Create a real array of length `n` of equidistantly-spaced elements between `lower` and `upper`
+
+<!-- vector; linspaced_vector; (int n, real lower, real upper); -->
+\index{{\tt \bfseries linspaced\_vector }!{\tt (int n, real lower, real upper): vector}|hyperpage}
+
+`vector` **`linspaced_vector`**`(int n, real lower, real upper)`<br>\newline
+Create an `n`-dimensional vector of equidistantly-spaced elements between `lower` and `upper`
+
+<!-- row_vector; linspaced_row_vector; (int n, real lower, real upper); -->
+\index{{\tt \bfseries linspaced\_row\_vector }!{\tt (int n, real lower, real upper): row\_vector}|hyperpage}
+
+`row_vector` **`linspaced_row_vector`**`(int n, real lower, real upper)`<br>\newline
```

Comment:
```suggestion
`row_vector` **`linspaced_row_vector`**`(int n, data real lower, data real upper)`<br>\newline
```

Reply:
Hey @rok-cesnovar , thanks for the quick review. All done.
Cheers,
Simon